pyvista/core/filters.py

```diff
@@ -2265,23 +2266,52 @@ def extract_surface(dataset, pass_pointid=True, pass_cellid=True, inplace=False)
             Adds a cell array "vtkOriginalPointIds" that idenfities which
             original cells these surface cells correspond to
 
+        subdivision : int, optional
+            If the input is an unstructured grid with nonlinear faces,
```

Comment:
Concerning my long-winded comment on the example: I guess this part is OK if we assume that the reader knows what "nonlinear faces" exactly entails. Arguably it's not the reference doc's job to teach basic VTK, so perhaps any further explanation is better left with the example.

Reply:
I'm afraid that it's going to be an "exercise left up to the reader".  VTK docs contain some decent explanation, Paraview has good examples with pictures, and for the rest there's google.  I've included a better explanation in the examples docs.